lib/rubocop/cli.rb

```diff
@@ -77,6 +77,8 @@ def handle_exiting_options
 
       puts RuboCop::Version.version(false) if @options[:version]
       puts RuboCop::Version.version(true) if @options[:verbose_version]
+
+      configuration
```

Comment:
This method invocation looks pretty odd - judging by the name it should have been invoked for its return value, but it's actually invoked because of its side-effects.